organizer-guide.Rmd

```diff
@@ -7,20 +7,43 @@ This event is a key event for professionals in the genomics community interested
-- Develop high interest topics
-- Open science
+- Build an inclusive community that is supportive of newcomers
+- Contribute to [open science](https://en.wikipedia.org/wiki/Open_science)
+- Produce a final product that accomplishes one of the following for a topic
+  - Makes useful forward progress
+  - Identifies opportunities and boundaries or limitations
+
+The final product of CoFests doesn't need to be code! These Collaboration Fests can support story-boarding future work or development of documentation or training materials. If it's something that will help the community and make progress on an AnVIL associated topic and there's sufficient interest, the idea is game! [As Galaxy phrased it for their CoFests:](https://galaxyproject.org/events/gcc2023/cofest/#goals)
+
+> [The goal of collaboration fests is to expand the ecosystem] Not just the code, but the whole ecosystem. That includes training, tools, best practice workflows, documentation, test cases, translations, infrastructure, and yes, even code.  
+
+Prior to the CoFests, the scientific and AnVIL co-leads will work together to enumerate some ideas for final products and post them publicly in this book.
 
 ## Logistics {- .unlisted}
 
-- Parallel tracks, 5+ hours across Nov 12/13
-- Scientific Lead and Co-Lead
-- ~4-10 participants per topic
+- 3 Parallel tracks, 5+ hours across Nov 12/13
+  - Scalable Machine Learning and Data Science
+  - Polygenic Risk Score Analysis
+  - Wrap a Tool (+ Intro AnVIL/WDL)
+- Scientific Lead and AnVIL Outreach Co-Lead for support
+- We expect 4-10 participants per track
+
+## Example Schedule for a Track {- .unlisted}
+
+Day 1 (2.5 hrs scheduled)
+
+| Task                      | Suggested time | Lead                  | Description |
+|:-------------------------:|:---------------|:----------------------|:------------|
+| Problem statement         | 20-30 minutes  | Scientific Lead       | Provide background info on the track's overall topic and may include a brief summary of the pre-worked collaboration ideas.|
+| Brainstorming             | 15-30 minutes  | Both                  | What project will the track tackle? Focus on understanding the participant backgrounds and skills and matching those with track ideas and actionable tasks to accomplish the goal. |
+| Onboarding                | 30-60 minutes  | AnVIL Outreach Co-lead| Handle billing accounts and other logistics necessary to accomplish the work |
+| Work-regroup, work-regroup| 30-60 minutes  | --                    | Formal collaboration work time left before dinner. (Groups may choose to work together informally after dinner too) |
+
+Day 2 (3 hrs scheduled)
 
-## Example Schedule {- .unlisted}
+| Task                      | Suggested time | Lead                  | Description |
+|:-------------------------:|:---------------|:----------------------|:------------|
+| Work-regroup, work-regroup| 2 - 2.5 hrs    | --                    | Formal collaboration work time (with a coffee break) |
```

Comment:
```suggestion
| Work-regroup | 2 - 2.5 hrs    | --                    | Formal collaboration work time (with a coffee break) |
```

Or is "Work-regroup, work-regroup" actually meant to be?

Reply:
It's a phrase that @cutsort used in our meeting and the original draft and I kept it